src/UI/Http/Rest/Controller/Event/GetEventsController.php

```diff
@@ -29,6 +30,9 @@ public function __invoke(Request $request): JsonResponse
         $page = $request->get('page', 1);
         $limit = $request->get('limit', 50);
 
+        Assertion::integer($page, 'Page number must be an integer');
+        Assertion::integer($limit, 'Limit results must be an integer');
+
         $query = new GetEventsQuery($page, $limit);
```

Comment:
Exactly that but casting to int here.
PHP will do that automatically but not a problem for explicit `GetEventsQuery((int) $page, (int) $limit)`
By that way we've now input validation so will not incur in type errors.